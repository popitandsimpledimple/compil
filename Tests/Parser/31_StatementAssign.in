var a : integer;
begin
    a := 1;
    a += 2;
    a -= 3;
    a *= 4;
    a /= 5;
end.