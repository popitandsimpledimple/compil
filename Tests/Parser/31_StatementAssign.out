program
   decls
      vars
         var
            a
            type
               integer
            no expression
   block
      compounds statements
         :=
            a
            1
         +=
            a
            2
         -=
            a
            3
         *=
            a
            4
         /=
            a
            5
